packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -68,7 +68,7 @@ export default decorate([
             username = remote.username,
             protocol = remote.protocol || 'https',
             region = remote.region,
-            allowUnauthorized = remote.allowUnauthorized,
+            allowUnauthorized = remote.allowUnauthorized === true ? true : undefined,
```

Comment:
I am not sure, but I prefer to be strict

Reply:
I'm not fond of testing strict equality with boolean unless absolutely necessary.

I don't want this change unless there is a specific reason to do it and a comment in the code explaining it, otherwise we might remove it in the future.